packages/scripts/src/commands/app-data/postProcess/sheets.ts

```diff
@@ -45,6 +45,7 @@ class SheetsPostProcessor {
     fs.emptyDirSync(appSheetsFolder);
     // Handle Copy
     this.sheetsCopyFiles(sourceSheetsFolder, appSheetsFolder);
+    this.hackGenerateCoreAssetContentsDataList(app_data.output_path, appSheetsFolder);
```

Comment:
nit(blocking)
- Would be good to refer to any methods related to asset_pack sheets with the term `assetPack` to disambiguate from assets
- As the asset pack sheet will need to be updated whenever assets are synced really it would be better to find a way to include this method as part of asset post-processing
- Currently the postProcess methods has support for tests, so would be good to include some specific tests related to this feature to cover core behaviour